### PR TITLE
[WIP] fix(facade): Array.forEach only support a single argument

### PR DESCRIPTION
**Pull request by vicb**
[Review on Reviewable](https://reviewable.io/reviews/angular/ts2dart/213)

### CODE REVIEW

**Comment by vicb** [comment]
WIP: I should add support for `A.forEach(fn)` (ie not using an inline fn but referencing a fn)

File: lib/facade_converter.ts [hunk 1]
```diff
@@ -174,6 +174,14 @@ export class FacadeConverter extends base.TranspilerBase {
       this.visit(c.arguments[0]);
       this.emit(')');
     },
+    'Array.forEach': (c: ts.CallExpression, context: ts.Expression) => {
+      const callback = <ts.FunctionExpression>c.arguments[0];
+      if (callback.parameters.length !== 1) {
+        this.reportError(c, 'Array.forEach callback supports a single argument only');
```
Comment (mprobst) [inline lib/facade_converter.ts]: nit: `takes exactly one argument`?

File: test/facade_converter_test.ts [hunk 2]
```diff
@@ -132,9 +136,18 @@ describe('type based translation', () => {
   });
 
   describe('error detection', () => {
+    describe('Array', () => {
+      it('.forEach should error on multiple arguments', () => {
+        chai.expect(() => translateSource('var x: Array<number> = []; x.forEach((v, i) => v);',
```
Comment (mprobst) [inline test/facade_converter_test.ts]: `expectErroneousWithType` ?

Comment (vicb) [inline test/facade_converter_test.ts]: Ah that was it means :)
I'll update.

**Pull request closed without merging**